### PR TITLE
Rust API Client library change

### PR DESCRIPTION
**Pull request by rmnoff**
Hi! Guys it is the time to change rust library, because the one ipfs has right now is undone and wasn't updated since 2015.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -182,7 +182,7 @@ If you would you like to start your own language implementation of IPFS, check o
 | Haskell     | https://github.com/davidar/hs-ipfs-api            | |
 | Swift       | https://github.com/ipfs/swift-ipfs-api            | |
 | CommonLisp  | https://github.com/WeMeetAgain/cl-ipfs-api        | |
-| Rust        | https://github.com/rschulman/rust-ipfs-api        | |
```
Comment (diasdavid) [inline README.md]: Could you just list both instead of replacing? See other examples.

Comment (rmnoff) [inline README.md]: check out last commit, is it right? :)

Comment (diasdavid) [inline README.md]: Almost, see my fix :)

Comment (rmnoff) [inline README.md]: thank you 👍

**Pull request merged**